keras/src/dtype_policies/dtype_policy.py

```diff
@@ -3,7 +3,7 @@
 from keras.src.api_export import keras_export
 from keras.src.backend.common import global_state
 
-QUANTIZATION_MODES = ("int8", "float8", "int4")
+QUANTIZATION_MODES = ("int8", "float8", "int4", "gptq")
```

Comment:
gptq would work with the other dtype modes correct? Wondering if adding the gptq mode here is appropriate or it needs to be a separate